### PR TITLE
DB-9035 Let SQLVarchar share SQLChar's hashCode() function

### PR DESCRIPTION
**Pull request by yxia92**

### CODE REVIEW

**Comment by yxia92** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLVarchar.java [hunk 1]
```diff
@@ -416,25 +416,7 @@ public BooleanDataValue greaterOrEquals(DataValueDescriptor left,
...
-				return 0;
-			} else {
-				return str.hashCode();
-			}
-		}
-		catch (StandardException se)
-		{
-			throw new RuntimeException(se);
-		}
+		return super.hashCode();
```
Comment (arnaud-splice) [inline db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLVarchar.java]: Why override it at all and not let it be inherited?

**Comment by changli6** [comment]
@yxia92 There are conflicts on merging, could you resolve it?

**Pull request merged**